src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -2171,16 +2171,14 @@ def call(
 
         # set global attention on question tokens
         if inputs["global_attention_mask"] is None and inputs["input_ids"] is not None:
-            if inputs["input_ids"] is None:
-                logger.warning(
-                    "It is not possible to automatically generate the `global_attention_mask`. Please make sure that it is correctly set."
```

Comment:
can we leave this warning?

Reply:
We can but here the problem is that we test if `inputs["input_ids"]` is None inside a if that already test if ` inputs["input_ids"]` is not None, this seems strange

Reply:
Agreed with @jplu here ;-)